### ISSUE TITLE
OnsenUIModule_getInstance()._on.platform is undefined

### ISSUE THREAD

**Issue by trixpan** [rjaros/kvision-examples#26]
not sure when this broke but today, when testing 5.8.3 I noticed a similar error. Tried downloading the template (to confirm it is not related to my own code) and encountered the same issue

![screen](https://user-images.githubusercontent.com/3108527/162662269-8fb7cadc-2a92-4e01-a915-bd224aa2c56b.jpg)

**Comment by rjaros** [comment]
Hello,
I can't reproduce the issue. How do you build and run this application? In the browser on your computer? On some other device? What platform and browser do you use?

**Comment by trixpan** [comment]
@rjaros 

inteliij > gradle run
Browsers are running on Linux, x86_64 , tested both on Chrome and Firefox. Both had the same result.

**Comment by rjaros** [comment]
Have you tried cleaning, building and running the app directly from the command line?

**Comment by trixpan** [comment]
tried removing

~/.gradle
<path_to_project>/build/

run with `./gradlew run` but same result. :-(

**Comment by rjaros** [comment]
How about production build (`./gradlew zip` and opening `build/distributions/index.html` in the browser)? Does it work for you? 
If not, can you send me `build/libs/onsenui-kitchensink-1.0.0-SNAPSHOT.zip` file?

**Comment by rjaros** [comment]
OK. I have reproduced the problem by removing my `/.kotlin-js-store/yarn.lock`. OnsenUI 2.12.0 was released yesterday and it's somehow not compatible with 2.11.2.

**Comment by rjaros** [comment]
The new version has a lot of breaking changes for a minor release ;-)
https://github.com/OnsenUI/OnsenUI/releases/tag/2.12.0

I think you will have to wait for new KVision release (should be released soon).
For now you can force OnsenUI 2.11 with this:

```Kotlin
rootProject.plugins.withType<org.jetbrains.kotlin.gradle.targets.js.yarn.YarnPlugin> {
    rootProject.the<org.jetbrains.kotlin.gradle.targets.js.yarn.YarnRootExtension>().apply {
        resolution("onsenui", "2.11.2")
    }
}
```

**Comment by trixpan** [comment]
> The new version has a lot of breaking changes for a minor release ;-)

ROFL. Gotta love dependencies. :-) 

Thanks for investigating (and providing workaround). I was starting to feel dumb. LOL.

PS-Tested workaround and can confirm it works

**Comment by rjaros** [comment]
OnsenUI support fixed in KVision 5.9.0

**Comment by trixpan** [comment]
tested upgrade and can confirm it was fixed
